pkg/arvo/sys/vane/ames.hoon

```diff
@@ -2379,9 +2379,13 @@
                 :_  tang.u.dud
                 leaf+"ames: {<her>} fragment crashed {<mote.u.dud>}"
             abet:(call:abed:(sink-core bone) %hear lane shut-packet ?=(~ dud))
-          ::  benign ack on corked bone
+          ::  benign ack on corked bone (also for naxplanation acks)
           ::
-          ?:  (~(has in corked.peer-state) bone)
+          ?:  ?|  (~(has in corked.peer-state) bone)
```

Comment:
Since this code is repeated, it should be factored out into an `+is-corked` gate.